src/components/SiteList.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import summary from '../apis/summary';
+
+import SiteItem from './SiteItem';
+
+class SiteList extends Component {
+  state = {
+    sites: []
+  };
+
+  componentDidMount() {
+    summary.get('/sites').then(response => {
+      this.setState({ sites: response.data.features });
+    });
+  }
+
+  siteSelectHandler(id) {
+    this.props.history.push({ pathname: '/' + id });
+  }
+
+  render() {
+    let sites = this.state.sites.map(site => {
+      return (
+        <div className="item" key={site.id}>
```

Comment:
Strongly advise that you keep away from using ID's in mapped series of elements whenever possible, just because we're using long strings. 

When you're declaring `map`, try using 
```js
sites = this.state.sites.map((site, key) => { 
 ...
 <div className="item" key={key}>
...
})
```

The key is an iterating integer that will always refer to the ordered number of that list. Key's don't fail, and are a wonderful part of why I use maps for building lists of react components!

Reply:
it makes sense, do you recommend using index then?

Reply:
hmm. I might be wrong here: https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

You should stick with using the `id` as opposed to the index of the map.